File: isegoria/src/main/java/com/eulersbridge/isegoria/models/Country.java
package com.eulersbridge.isegoria.models;

import com.squareup.moshi.Json;

import java.util.List;

public class Country {

	@Json(name = "countryId")
	public long id;

	@Json(name = "countryName")
	public String name;
	public List<Institution> institutions;

	public Country(String name) {
	    this.name = name;
    }

}

File: isegoria/src/main/kotlin/com/eulersbridge/isegoria/network/api/model/Country.kt
package com.eulersbridge.isegoria.network.api.model

import com.squareup.moshi.Json

data class Country(
    @field:Json(name = "countryId")
    val id: Long = 0,

    @field:Json(name = "countryName")
    val name: String,

    val institutions: List<Institution>?
) {
    override fun toString() = name
}